srcpkgs/vala-panel/template

```diff
@@ -1,20 +1,25 @@
 # Template file for 'vala-panel'
 pkgname=vala-panel
-version=0.4.88
+version=0.5.0
 revision=1
-build_style=cmake
-hostmakedepends="gettext pkg-config vala glib-devel"
-makedepends="vala-devel cmake-vala gtk+3-devel"
-short_desc="GTK+3 desktop panel inspired by LXPanel"
-maintainer="John Eira <john.eira@web.de>"
+build_style=meson
+configure_args="$(vopt_if wayland -Dplatforms=x11,layer-shell -Dplatforms=x11)"
+hostmakedepends="vala pkg-config cmake gettext glib-devel"
+makedepends="gtk+3-devel libwnck-devel $(vopt_if wayland gtk-layer-shell-devel)"
+short_desc="GTK+3 desktop panel forked from LXPanel"
+maintainer="Joshua Krämer <joshua@kraemer.link>"
```

Comment:
Have you contacted the previous maintainer to see if they are ok with this? @purpurrot

Reply:
I wrote an email to him on 2020-11-07, but he did not answer. He also had no activity in his GitHub account during the last year. I thus assumed that he is no longer interested in maintaining the package.

Reply:
I wrote an email to him on 2020-11-07, but he did not answer. Additionaly, there was no activity on his GitHub account during the last year. I thus assumed that he is no longer interested in maintaining the package.

Reply:
Ok, that's fair.